app/controllers/api/v1/ability.rb

```diff
@@ -180,6 +187,7 @@ def image_abilities
             record.imageable.offer.created_by_id == @user_id
           end
           can [:show], Image, { imageable_type: "Package", imageable: { order: { created_by_id: @user_id } } }
+          can :create, Image, { imageable_type: nil }
```

Comment:
@swatijadhav Possible to change to ?

`can :create, Image, { imageable_type: 'User', imageable_id: @user_id }`

Reply:
As per our last discussion it is going to be association change and will reflect in all apps on front-end too and will need few more days to work on it.